packages/php-wasm/universal/src/lib/base-php.ts

```diff
@@ -616,6 +616,39 @@ export abstract class BasePHP implements IsomorphicLocalPHP {
 		this[__private__dont__use].FS.rename(fromPath, toPath);
 	}
 
+	/** @inheritDoc */
+	@rethrowFileSystemError('Could not copy "{path}"')
+	cp(fromPath: string, toPath: string) {
+		const FS = this[__private__dont__use].FS;
+
+		const fromStat = FS.stat(fromPath);
+
+		const fromMode = parseInt(fromStat.mode.toString(8).substring(0, 1));
+
+		const fromIsDir = fromMode === 4;
+
+		if (fromIsDir) {
+			try {
+				FS.mkdir(toPath);
+			} catch {
+				// Do nothing
+			}
+
+			const files = FS.readdir(fromPath);
+
+			files
+				.filter((f: string) => !['.', '..'].includes(f))
+				.forEach((f: string) =>
```

Comment:
Let's avoid single-letter variable names and use full words. Looking around the file and seeing just `f` is quite confusing and requires tracing the code back to the line that says `files` and then keeping that context in mind.

Reply:
@adamziel Done. https://github.com/WordPress/wordpress-playground/pull/846/files#diff-b33e1e6834fb94ac2cb5338e3ae17c60abf7ac88cb057189a1915531e55f7243R655